### PR TITLE
Added TabBarIOS and TabBarItemIOS

### PR DESCRIPTION
**Pull request by lpalmes**
Added missing components for my own use and i thought they may come in handy for somebody else.

### CODE REVIEW

File: src/components/tabBarIOSRe.rei [hunk 1]
```diff
@@ -0,0 +1,10 @@
+let make:
+  barTintColor::'a? =>
```
Comment (saschatimme) [inline src/components/tabBarIOSRe.rei]: Could you fix the types here?

File: src/components/tabBarItemIOSRe.rei [hunk 2]
```diff
@@ -0,0 +1,13 @@
+let make:
+  title::string? =>
+  badgeColor::string? =>
+  badge::'a? =>
```
Comment (saschatimme) [inline src/components/tabBarItemIOSRe.rei]: here also

**Comment by saschatimme** [comment]
Really cool! We just need to fix the `.rei`s and we are good to got :)

**Comment by lpalmes** [comment]
I've added types, one thing that i found out is that it might be a good idea to extract imageSource from the ImageRe module to have other modules use it like this one, let me know your opinion about it. Thanks!

**Comment by saschatimme** [comment]
This looks good now :)
I think it would make sense to extract imageSource but we should do this separate since this occurs on other places as well. I will open a separate issue to discuss the best approach.

**Pull request merged**